lib/semian/simple_state.rb

```diff
@@ -27,7 +27,7 @@ def close
         @value = :closed
       end
 
-      def half_open
+      def half_open!
```

Comment:
When there's two methods of the same name (even if they have different `?`/`!`) suffixes—I like to name the more dangerous one `!`.